src/components/users/add-member-group-dialog.tsx

```diff
@@ -59,12 +70,14 @@ const AddMemberGroupDialog: React.FC<Props> = ({
         />
       </DialogContent>
       <DialogActions style={{ padding: 24, paddingTop: 0 }}>
-        <Button disableElevation variant="contained" onClick={ onCancel } color="secondary" autoFocus>
+        <Button disableElevation variant="contained" onClick={ onCancelClick } color="secondary" autoFocus>
           { strings.userManagementScreen.addMemberGroupDialog.cancelButton }
         </Button>
-        <Button onClick={ onCreateClick } color="primary" disabled={ !displayName }>
-          { strings.userManagementScreen.addMemberGroupDialog.createButton }
-        </Button>
+        <GenericLoaderWrapper loading={ loading }>
```

Comment:
Maybe we could just disable the button when something is loading instead of using loader?